__tests__/app.test.js

```diff
@@ -69,3 +69,42 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("PATCH /api/articles/:article_id", () => {
+  test("Status:200 request body accepts an object with a key/value pair which holds the number to increment the votes property and returns the updated object", () => {
+    const requestObj = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(requestObj)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          article_id: 1,
+          title: expect.any(String),
+          topic: expect.any(String),
```

Comment:
We can be even more explicit than this because we know exactly what article 1 looks like from the database!
Please replace with exact values.